docs/accuracy_aware_model_training/AdaptiveCompressionLevelTraining.md

```diff
@@ -1,6 +1,17 @@
 # Adaptive Compression Level training loop in NNCF
 
-The search can be only done for a single compression algorithm in the pipeline (i.e. several compression algorithms could be applied to the model and the search is going to be performed for a single one); currently supported algorithms for compression rate search are magnitude sparsity and filter pruning. The exact compression algorithm for which the search is done is determined from `"accuracy_aware_training"` config section added to the target compression algorithm section. Below is an example of a filter pruning configuration with added `"accuracy_aware_training"` parameters. The parameters to be set by the user in this config section are (i) `maximal_relative_accuracy_degradation` - the maximal allowed accuracy metric drop relative to the original model (in percent), (ii) `initial_training_phase_epochs` - the number of epochs to train the model with the compression schedule specified in the  `"params"` section, (iii) `patience_epochs` - the number of epochs to train the model for a compression rate level set by the search algorithm before switching to another compression rate value.
+The compression level search can be only done for a single compression algorithm in the pipeline (i.e. several compression algorithms could be applied to the model and the search is going to be performed for a single one); currently supported algorithms for compression rate search are magnitude sparsity and filter pruning. 
```

Comment:
Let's use capital letters for magnitude sparsity and filter pruning.

Reply:
I think it's confusing or I misunderstand it: can it be used for Pruning+INT8 for instance?

Reply:
Update with examples. Please have a look